styles/colors-cobalt-day.less

```diff
@@ -271,4 +271,4 @@
 
 // Scroller
 // ---------------------------------------
-@agate-scroller-scrollbar-thumb-bg-color: #373737;
+@agate-scroller-scrollbar-thumb-bg-color: @agate-accent;
```

Comment:
Try to find a common variable for all skins. How about @agate-text-color on all skins? If it looks alright, then you can use it in colors-base and remove @agate-scroller-scrollbar-thumb-bg-color from colors-[skin].less

Reply:
I used `@agate-text-color` as base color, and in my opinion it looks good. For now on Copper skin it does not look so great, but after [#564 ](https://github.com/enactjs/agate/pull/564) is merged, it should be ok.